changelog/709.misc.rst

```diff
@@ -0,0 +1,4 @@
+pass the ``PROGRAMDATA`` environment variable when invoking the internal
```

Comment:
this is a development environment change for tox, we don't need a changelog entry for this as servers no value for end user